pkg-manager/core/src/install/extendInstallOptions.ts

```diff
@@ -48,6 +48,7 @@ export interface StrictInstallOptions {
   lockfileIncludeTarballUrl: boolean
   preferWorkspacePackages: boolean
   preserveWorkspaceProtocol: boolean
+  requiresbuildFromRegistry: boolean
```

Comment:
This is very specific to npmjs, so I'd say use:
use-experimental-npmjs-files-index